.github/workflows/preDeploy.yml

```diff
@@ -157,6 +160,33 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
           NPM_VERSION: ${{ env.NEW_VERSION }}
 
+      - name: Comment in StagingDeployCash to alert Applause that a new pull request has been cherry-picked
+        if: ${{ env.DO_CHERRY_PICK }}
+        run: |
+          PR_URL="https://github.com/Expensify/App/pull/${{ needs.chooseDeployActions.outputs.mergedPullRequest }}"
+          printf -v COMMENT ":clap: Heads up @Expensify/applauseleads :clap:\nA [new pull request](%s) has been :cherries: cherry-picked :cherries: to staging, and will be deployed to staging in version \`%s\` :rocket:" "$PR_URL" ${{ env.NEW_VERSION }}
+          gh issue comment \
+          "$(gh issue list --label StagingDeployCash --json number --jq '.[0].number')" \
+          --body "$COMMENT"
+        env:
+          GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
+
+      - name: Wait for staging deploys to finish
+        if: ${{ env.DO_CHERRY_PICK }}
+        uses: Expensify/App/.github/actions/awaitStagingDeploys@main
+        with:
+          GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
+          TAG: ${{ env.NEW_VERSION }}
+
+      - name: Comment in StagingDeployCash to alert Applause that cherry-picked pull request has been deployed.
+        if: ${{ env.DO_CHERRY_PICK }}
+        run: |
+          gh issue comment \
+          "$(gh issue list --label StagingDeployCash --json number --jq '.[0].number')" \
+          --body ":tada: All set?…You bet! https://github.com/Expensify/App/releases/tag/${{ env.NEW_VERSION }} has been deployed to staging :tada:"
```

Comment:
Should this actually tag Applause leads? I don't think it does at the moment?

Reply:
Fixed!